testing/model/test_dbmodel.py

```diff
@@ -34,8 +34,8 @@ def __init__(self, uri: str):
     def setup(self):
         if self.uri.startswith("mysql"):
             # FIXME: Use `host`, `user`, `password`, and `dbname` from DB URI.
-            dbname = "secpi-testdrive"
-            conn = pymysql.connect(host="localhost", user="root", password="secret")
+            dbname = "secpi_test"
+            conn = pymysql.connect(host="localhost", user="secpi", password="secret")
```

Comment:
Let's stick with `user="secpi"`. Thank you. The Docker configuration can be adjusted accordingly.

```
MARIADB_USER: secpi
MARIADB_PASSWORD: secret
```
-- https://github.com/isarengineering/SecPi/blob/next/.github/workflows/tests.yml#L35 

```
--env "MARIADB_USER=secpi" --env "MARIADB_PASSWORD=secret"
```

-- https://github.com/isarengineering/SecPi/blame/next/doc/sandbox.rst#L20

Reply:
Works. Thanks.